common/config/babel/.babelrc.js

```diff
@@ -58,8 +58,19 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
         // feature for blurred background and replace background effects
         'video-background-effects',
         // feature for vertical gallery layouts in VideoGallery
-        'vertical-gallery'
+        'vertical-gallery',
+        // a demo feature flag for those beta feature not ready for beta release
+        'in-progress-beta-feature-demo'
+      ],      
+      // A list of in progress beta feature.
+      // These features are still beta feature but "in progress"
+      // causing a build failure, but they are ignored by the preprocessing step.
+      inProgressFeatures: [
```

Comment:
Take a note here from offline chat with Donald - this could be named as 'alpha' or something to have a more straightforward meaning

Will bring this up with our whole team when we talk about the proposal